tailwind.config.js

```diff
@@ -0,0 +1,62 @@
+const path = require("path");
+const decanter = path.resolve(__dirname, "node_modules/decanter/src/plugins");
+
+module.exports = {
+  mode: "jit",
+  prefix: "su-",
+  purge: [
+    "./src/**"
+  ],
+  darkMode: false, // or 'media' or 'class'
+  theme: {
+    borderRadius: require(decanter + "/theme/borderRadius.js")(),
```

Comment:
Would globbing help with any/all of these require statements? Right now it is a very manual process of adding and removing files as they come and go and this could be a problem for upgrade paths down the line. Perhaps globbing the files might be better?

Reply:
Globbing and dynamical loading has been added.